lib/collectionspace/client/service.rb

```diff
@@ -12,6 +12,12 @@ def self.get(type:, subtype: "")
           path: "acquisitions",
           term: nil
         },
+        "chronologyauthorities" => {
+          identifier: "shortIdentifier",
+          ns_prefix: "chronologies",
+          path: "chronologyauthorities/urn:cspace:name(#{subtype})/items",
+          term: "chronologie#{TERM_SUFFIX}"
```

Comment:
Is it really `chronologie` not `chronology`? -- looks weird 😄

Reply:
Sigh. Nope. Because no pattern can ever hold in CS, the mapper is wrong. I'll fix this in a sec and the mappers...